### PR TITLE
Create mdwasim74.jason

### PR DESCRIPTION
**Pull request by mdwasim74**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8762"></a>

### CODE REVIEW

File: Data/mdwasim74.jason
Comment (sital002) [inline Data/mdwasim74.jason:5]:
```suggestion
   "links" : [
```

**Review by sital002 — CHANGES_REQUESTED** [review]
Your file must be in data folder instead of ``` Data/mdwasim74.jason ``` It must be in ``` data/mdwasim74.json ```.

**Comment by SaraJaoude** [comment]
> Your file must be in data folder instead of `Data/mdwasim74.jason` It must be in `data/mdwasim74.json`.

@sital002 good spot.  I will add this as a requested change so that it blocks the merge (and prevents this PR being merged by mistake without the changes being addressed).

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please move your file to the `data` folder. Instead of `Data/mdwasim74.jason` It must be in `data/mdwasim74.json`.

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart-json to help you update this Pull Request.

**Comment by SaraJaoude** [comment]
I will close this PR as it appears to be a duplicate of https://github.com/EddieHubCommunity/LinkFree/pull/8767 and we will take your latest PR with the most recent changes as the one you want to keep. 

In future, to reduce extra work for maintainers, please either edit (you might find [these Docs](https://linkfree.io/docs/how-to-guides/editing-json) helpful) or close PRs that you no longer want.

**Pull request closed without merging**
